### PR TITLE
Update known-issue-178-multi-role-rls-composite-model-fail.md

### PR DESCRIPTION
**Pull request by chaiyiding**
Resolve the issue as fix has been deployed

### CODE REVIEW

**Comment by Jak-MS** [comment]
@mihart 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by chaiyiding** [comment]
@mihart @jeroenterheerdt can you review and approve the pr?

#sign-off

File: powerbi-docs/troubleshoot/known-issues/known-issue-178-multi-role-rls-composite-model-fail.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ If you face this issue, you will see an error similar to:
 
 ## Solutions and workarounds
 
-In some cases, you may be able to combine RLS roles to circumvent this error. But in most cases, there is no workaround at this time. When the fix is released, this article will be updated.
+Fix has been deployed and the above error has been removed. User that belongs to multiple roles for composite models will experience the same behavior as user that belongs to multiple roles for non-composite model: the user will have access to all the data that each individual role has access to. For example if role 'A' has access to Product[Color] = "Black" and role 'B' has accesso to Product[Color] = "Red", a user belongs to both role 'A' and 'B' will have acess to Product[Color] = "Black" or "Red".
```
Comment (maggiesMSFT) [inline powerbi-docs/troubleshoot/known-issues/known-issue-178-multi-role-rls-composite-model-fail.md:31]:
```suggestion
Fix has been deployed and the above error has been removed. Users who belong to multiple roles for composite models will experience the same behavior as users who belong to multiple roles for non-composite models: they'll have access to all the data that each individual role has access to. For example, if role 'A' has access to Product[Color] = "Black" and role 'B' has access to Product[Color] = "Red", a user who belongs to both role 'A' and 'B' will have access to Product[Color] = "Black" or "Red".
```

**Review by maggiesMSFT — APPROVED** [review]

**Comment by maggiesMSFT** [comment]
#sign-off

**Pull request merged**